qa/tasks/cephfs/test_admin.py

```diff
@@ -1818,6 +1818,15 @@ def test_single_path_authorize_on_nonalphanumeric_fsname(self):
         self._remount(keyring)
         self.captester.run_mds_cap_tests(PERM)
 
+    def test_fs_read_and_single_path_rw(self):
+        FS_AUTH_CAPS = (('/', 'r'), ('/dir2', 'rw'))
+        self.mount_a.run_shell('mkdir -p ./dir2')
+        self.captesters = (CapTester(self.mount_a, '/'),
+                           CapTester(self.mount_a, '/dir2'))
+        keyring = self.fs.authorize(self.client_id, FS_AUTH_CAPS)
+
+        self._remount_and_run_tests(FS_AUTH_CAPS, keyring)
```

Comment:
Does this also check that the client does not have `w` access to `/` (possibly given accidentally by a bug where the grant on `/dir2` leaks)?

Please add a test description at the beginning.

Reply:
Yes, it does at https://github.com/ceph/ceph/pull/58878/commits/338b83b03169bff4ac821d207aa7414d70bb2de0#diff-9998476d9d02f4136e9f6b4b512be93a77a5911a30027bec6a120706a1a024f3R271

The missing part was to test `setattr` when it has `rw` caps. The new file creation using `touch` should cover it.

Reply:
Added the test decription

Reply:
I don't see it?

Reply:
My bad, some mess up while doing `git add`. Should be visible now.